core/lib/spree/core/mail_method.rb

```diff
@@ -4,9 +4,9 @@ class MailMethod
       def initialize(options={})
       end
 
-      def deliver!(mail)
+      def deliver_now!(mail)
         if Config.enable_mail_delivery
-          mailer.deliver_now!(mail)
+          mailer.deliver!(mail)
```

Comment:
I think this should still be able to use `deliver_now!` as it is the method in 4-2